src/ax/saga/workflow.go

```diff
@@ -22,30 +22,33 @@ type Workflow struct {
 
 // NewWorkflow returns a saga that forwards to the given aggregate.
 //
-// Workflows are a specialization of sagas that handle events and produce
-// commands.
+// Workflows are a specialization of sagas that handle commands and/or events
+// and produce commands.
 //
 // It accepts a prototype data instance which is cloned for new instances.
 //
-// For each event type to be handled, the aggregate must implement a "handler"
+// For each message type to be handled, the aggregate must implement a "handler"
 // method that adheres to one of the following signatures:
 //
-//     func (ev *<T>, ax.CommandExecutor)
-//     func (ev *<T>, env ax.Envelope, ax.CommandExecutor)
+//     func (m *<T>, ax.CommandExecutor)
+//     func (m *<T>, env ax.Envelope, ax.CommandExecutor)
 //
-// Where T is a struct type that implements ax.Event.
+// Where T is a struct type that implements ax.Message.
 //
 // Handler methods are responsible for mutating the state of the workflow and
-// producing new commands, based on the event being handled. They may inspect
-// the current state of the workflow, and then return zero or more commands
-// to be executed.
+// producing new commands, based on the command or event being handled. They may
+// inspect the current state of the workflow, and then return zero or more
+// commands to be executed.
```

Comment:
Could you change this paragraph to simply read:

> Handler methods are responsible for mutating the state of the workflow and producing new commands, based on the message being handled.

The second sentence seems a bit redundant, not really related to your PR, but may as well fix it up since you're changing it already.